client/js/lounge.js

```diff
@@ -371,6 +371,7 @@ $(function() {
 
 		lastActiveChan
 			.find(".unread-marker")
+			.data("unread-id", 0)
```

Comment:
This is an id, not a counter, correct? If so, any risk of the id actually being `0` and leading to weird bugs? Wondering if we couldn't use `-1` or `null` or similar exceptional value instead then.

Reply:
It checks for `> 0` when using this id. If the message is 0, there can't be any older messages by definition, so nothing will happen.